### ISSUE TITLE
Support for SUPEE-6788

### ISSUE THREAD

**Issue by sergeifilippov** [jayelkaake/enhancedgrid#15]
Hi, Looks like this module will break with the pending release of SUPEE-6788 patch.

<img width="445" alt="1__filips01_orion____sites_magento_src__zsh__and_enhancedgrid_ _config_xml_-_src_-____sites_magento_src_" src="https://cloud.githubusercontent.com/assets/123676/10710016/7f955a2a-7aa0-11e5-888f-f0d492bd700e.png">

**Comment by jayelkaake** [comment]
Can you submit a PR? :dancers:

**Comment by jayelkaake** [comment]
Fixed! Just pull latest (1.3.4.1) with modman, composer or git.

**Comment by danemacmillan** [comment]
Can you update tags for http://packages.firegento.com/#!/enhancedgrid ?

**Comment by jayelkaake** [comment]
Sure! Will do. I've been having trouble updating Magento Connect (since the
credentials have changed).

Jay

**Comment by jayelkaake** [comment]
K this has been updated in all places. Firegento will take a bit of time to update, but it'll be there in both places soon.

**Comment by danemacmillan** [comment]
Awesome, thanks--will it be `1.3.4.2`?

**Comment by jayelkaake** [comment]
Yep!

**Comment by danemacmillan** [comment]
@jayelkaake I don't think it was done correctly. Firegento has updated at least a dozen times since Monday.

**Comment by jayelkaake** [comment]
I just needed to update the module version in github right? I thought it
would pick that up? I also updated the version in Magento Connect. Any idea
why it wouldn't pick it up?

J

**Comment by danemacmillan** [comment]
> Hint: Satis recognizes only annotated tags (git tag -a) as versions.

Source: https://github.com/magento-hackathon/composer-repository/blob/master/README.md

In other words, if you did not annotate the tag, you've essentially created what's called a lightweight tag. Annotated tags are needed for releases. See `git help tag`:

> Tag objects (created with -a, -s, or -u) are called "annotated" tags; they contain a creation date, the tagger name and e-mail, a tagging message, and an optional GnuPG signature. Whereas a "lightweight" tag is simply a name for an object (usually a commit object). 

> Annotated tags are meant for release while lightweight tags are meant for private or temporary object labels. For this reason, some git commands for naming objects (like git describe) will ignore lightweight tags by default.

**Comment by jayelkaake** [comment]
I used github to create the release that creates tags so I didn't think I and to do that. Composer recognizes this normally. Also I updated the module on Magento Connect and it's not updating on the list... hmm...Magento Connect was updated recently, wonder if that has to do with it...

Anyways, I deleted the tag manually in my console and repushed it using the -a flag. https://github.com/jayelkaake/enhancedgrid/releases Let's see if this works in an hour or so.

**Comment by danemacmillan** [comment]
Excellent, it shows up now on the non-connect20 section. It oddly removes the previous tag, which I think was `v1.3.2.2`, so anyone who may have been depending on that will no longer be able to reference it.

**Comment by jayelkaake** [comment]
Yeah.. . I don't know why the firegento module doesn't just look at the
github tags... And I don't know what to do about the Magento Connect
mobile...
Hmm...

Well at least we got something.

Jay
